src/components/album/AlbumChatBox.js

```diff
@@ -78,6 +78,23 @@ const AlbumChatBox = ({ spotifyUrl }) => {
     };
   }, [albumId, socket, loggedInUserId]);
 
+  useEffect(() => {
+    const fetchRecentMessages = async () => {
+      if (albumId && !socket) {
+        const response = await fetch(
+          `${process.env.REACT_APP_API_BASE_PATH}/chat/${albumId}`
+        );
+        const data = await response.json();
```

Comment:
We should prefer to use `axiosInstance` from the `apis` folder. It's fine overall to use fetch but:
1. we want to try to stick to 1 consistent approach throughout the app. Elsewhere in the app we're using axios, so better to continue following that practice
2. Our axios instance is set up with the BASE_PATH already, so it lets us not need to add it to the url everywhere
3. Our axios instance is set up to log errors and send a cookie if the user is signed-in so that if the backend route requires a logged in user, we don't have to do any extra work on the frontend code.